### ISSUE TITLE
[BUG] Bugs I encountered when uploading from android and browsing the website on pc

### ISSUE THREAD

**Issue by JVT038** [immich-app/immich#645]
So first of all, thanks for making this, I think it has a lot of potential to be a competitor for Google photos and/or Synology photos.
**Describe the bug**
1: When I switch apps (or just click on the overview button that shows all the opened apps), the progress percentage jumps back from whatever it was to 0%, and it just restarts...

2: For some unknown reason, a couple of files aren't being uploaded. I don't know, but when they hit 100%, Immich moves on to the next file, but the file doesn't appear on the website or the mobile app.

3: When a file has finished uploading, the counter of total amount of uploaded files doesn't change, nor does the remaining files counter. Perhaps this has something to do with 2?

4: The progress bar regularly shrinks a little bit and expands a little bit, depending on the percentage. So when it goes from 9% to 10%, 1 digit is added and the progress bar shrinks a little bit. Also, some numbers take a bit more space than others and the progress bar constantly expands and shrinks, based on this percentage. Not really a big problem, but it looks annoying, so could it be possible for the progress bar to have a fixed width that allows sufficient space for all percentages from 0% to 100%?

5: This last one is regarding the website on pc. Sometimes, when I scroll really fast, the images don't load instantly and it shows a white panel for a second or so (and that's ok). But it also shows the item ID or something like that? Screenshot down below.

**Task List**

*Please complete the task list below. We need this information to help us reproduce the bug or point out problems in your setup. You are not providing enough info may delay our effort to help you.*

- [x] I have read thoroughly the README setup and installation instructions.
- [x] I have included my `docker-compose` file.
- [x] I have included my redacted `.env` file.
- [x] I have included information on my machine, and environment.

**Screenshots**
![image](https://user-images.githubusercontent.com/47184046/189216668-69928643-e3f2-4e8a-b9a8-9333d46908f9.png)
So if you look closely in this image, you can see the gutter between the panels, and it shows the item IDs. Is it possible to have a temporary icon / image that shows instead of the ID? 

**System**
- Phone OS [iOS, Android]: `Android 11` (without Google Play Services)
- Server Version: `1.27.0`
- Mobile App Version: `1.27.0`

**Additional context**
`.env`:
```

DB_HOSTNAME=*****
DB_USERNAME=*****
DB_PASSWORD=*****
DB_DATABASE_NAME=****
REDIS_HOSTNAME=immich_redis
UPLOAD_LOCATION=/path/to/upload
JWT_SECRET=****
ENABLE_MAPBOX=false
MAPBOX_KEY=*****
```

**Comment by alextran1502** [comment]
1: When I switch apps (or just click on the overview button that shows all the opened apps), the progress percentage jumps back from whatever it was to 0%, and it just restarts...
**This is expected for foreground upload as it kills the uploading process when you put the app in the background.**

2: For some unknown reason, a couple of files aren't being uploaded. I don't know, but when they hit 100%, Immich moves on to the next file, but the file doesn't appear on the website or the mobile app.
**There is a queuing system to handle the post-processing jobs for the file, so it might take a bit depending on which file type it is**

3: When a file has finished uploading, the counter of total amount of uploaded files doesn't change, nor does the remaining files counter. Perhaps this has something to do with 2?
**This is strange, is there a way to reproduce it?**

4: The progress bar regularly shrinks a little bit and expands a little bit, depending on the percentage. So when it goes from 9% to 10%, 1 digit is added and the progress bar shrinks a little bit. Also, some numbers take a bit more space than others and the progress bar constantly expands and shrinks, based on this percentage. Not really a big problem, but it looks annoying, so could it be possible for the progress bar to have a fixed width that allows sufficient space for all percentages from 0% to 100%?
**Got it**

5: This last one is regarding the website on pc. Sometimes, when I scroll really fast, the images don't load instantly and it shows a white panel for a second or so (and that's ok). But it also shows the item ID or something like that? Screenshot down below.
**I assume you use Firefox? This is expected on Firefox as the way it handles unloaded images. Can you create a separate issue for this?**

**Comment by JVT038** [comment]
>  This is expected for foreground upload as it kills the uploading process when you put the app in the background.
But I enabled background backup, so shouldn't the uploading process continue in the background? 
>  This is strange, is there a way to reproduce it?
I don't know, it just happens for some unknown reason. I'll check out the file and see if I can send it to you (if it's not a sensitive video)

And I made a separate issue (#646), but you already saw that.

**Comment by alextran1502** [comment]
> > This is expected for foreground upload as it kills the uploading process when you put the app in the background.
> 
> But I enabled background backup, so shouldn't the uploading process continue in the background?
> 
> > This is strange, is there a way to reproduce it?
> 
> I don't know, it just happens for some unknown reason. I'll check out the file and see if I can send it to you (if it's not a sensitive video)
> 
> And I made a separate issue (#646), but you already saw that.

Background backup will start automatically on its own after you enable it. It has its own scheduler and detection mechanism

**Comment by alextran1502** [comment]
Closed as the issue need to be resolved is open in #646
